aten/src/ATen/native/cuda/PowKernel.cu

```diff
@@ -151,7 +169,15 @@ void pow_tensor_scalar_kernel_impl(TensorIterator& iter,
 }
 
 void pow_tensor_scalar_kernel(TensorIterator& iter, Scalar exp_scalar) {
-  if (isFloatingType(iter.dtype()) || exp_scalar.isIntegral(false)) {
+  if (isComplexType(iter.dtype())) {
```

Comment:
should the condition here be `isComplexType(iter.dtype()) || exp_scalar.isComplex(true)` ?

Reply:
Good catch!! ☺️